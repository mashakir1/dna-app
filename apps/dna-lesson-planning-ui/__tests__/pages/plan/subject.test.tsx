import { render } from "@testing-library/react";

import Page from "pages/plan/subject";

// TODO: WRITE TESTS TO HANDLE PAGE ON CLICKS ON BREADCRUMB AND ON SUBJECT CHOICE
describe.skip("Plan by subject page", () => {
  it("renders and matches snapshot", () => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
