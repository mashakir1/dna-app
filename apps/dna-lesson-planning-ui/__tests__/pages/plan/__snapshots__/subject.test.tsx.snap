// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plan by subject page renders and matches snapshot 1`] = `
<div>
  <div
    class="page"
  >
    <div
      class="header flex justify-between items-center"
    >
      <nav
        class="breadcrumbs flex"
      >
        <p
          aria-label="choose-unit"
          class="small breadcrumb active "
        >
          choose-unit
        </p>
        <p
          aria-label="journey"
          class="small breadcrumb  "
        >
          journey
        </p>
        <p
          aria-label="plan-lesson"
          class="small breadcrumb  "
        >
          plan-lesson
        </p>
        <p
          aria-label="review"
          class="small breadcrumb  "
        >
          review
        </p>
      </nav>
      <span>
        cancel
      </span>
    </div>
    <div
      class="container flex column gap-l"
    >
      <div
        class="flex column gap-m"
      >
        <button
          class="emptyIconButton undefined flex items-center gap-s"
          type="button"
        >
          <svg
            fill="none"
            height="20"
            viewBox="0 0 20 20"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M9.18468 2L10.5066 3.48721L4.1141 9.16947H20V11.1593H4.11461L10.5066 16.8411L9.18468 18.3283L0 10.1642L9.18468 2Z"
              fill="#30CCD6"
              fill-rule="evenodd"
            />
          </svg>
          <span>
            back
          </span>
        </button>
        <div
          class="flex justify-between items-center"
        >
          <div
            class="flex column gap-s"
          >
            <h4>
              choose-elements-title
            </h4>
            <p
              class="small"
            >
              choose-elements-subtitle
            </p>
          </div>
        </div>
      </div>
      <div
        class="flex gap-m wrap"
      />
      <div
        class="footer flex gap-m items-center"
      >
        <div
          class="durationCounter flex column justify-center gap-xs items-center"
        >
          <div
            class="flex gap-xs items-center"
          >
            <p
              class="large"
            >
              <b />
            </p>
            <p
              class="small"
            >
              minute-symbol
            </p>
          </div>
          <svg
            fill="none"
            height="24"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M12 4.63636C7.93318 4.63636 4.63636 7.93318 4.63636 12C4.63636 16.0668 7.93318 19.3636 12 19.3636C16.0668 19.3636 19.3636 16.0668 19.3636 12C19.3636 7.93318 16.0668 4.63636 12 4.63636ZM3 12C3 7.02944 7.02944 3 12 3C16.9706 3 21 7.02944 21 12C21 16.9706 16.9706 21 12 21C7.02944 21 3 16.9706 3 12ZM12 6.27273C12.4519 6.27273 12.8182 6.63904 12.8182 7.09091V11.4943L15.6386 12.9046C16.0428 13.1066 16.2066 13.5981 16.0045 14.0023C15.8024 14.4064 15.311 14.5702 14.9068 14.3682L11.6341 12.7318C11.3569 12.5932 11.1818 12.3099 11.1818 12V7.09091C11.1818 6.63904 11.5481 6.27273 12 6.27273Z"
              fill="#30CCD6"
              fill-rule="evenodd"
            />
          </svg>
        </div>
        <div
          class="selectedElementsList flex gap-s"
        />
        <div
          class="footerButtonList flex column gap-s justify-center"
        >
          <span>
            continue
          </span>
          <span>
            edit-slides
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
